src/utils.rs

```diff
@@ -605,14 +605,17 @@ pub fn format_url(url: &str) -> String {
 	}
 }
 
-// Rewrite Reddit links to Libreddit in body of text
+// Rewrite Reddit links to Libreddit in body of text.
+// Also fix superfluous backslashes and underscores in links.
 pub fn rewrite_urls(input_text: &str) -> String {
-
 	let text1 =
-		Regex::new(r#"href="(https|http|)://(www\.|old\.|np\.|amp\.|)(reddit\.com|redd\.it)/"#)
+		Regex::new(r#"href="(https|http|)://(www\.|old\.|np\.|amp\.|new\.)(reddit\.com|redd\.it)/"#)
+			// Replace reddit-prefix in href with "/" (i.e. point to our local instance).
 			.map_or(String::new(), |re| re.replace_all(input_text, r#"href="/"#).to_string())
-			// Remove (html-encoded) "\" from URLs.
-			.replace("%5C", "").replace(r"\", "");
+			// Remove (html-encoded) "\" from URLs. These seem to occur inside the href-attribute only.
+			.replace("%5C", "")
+			// Format underscores nicely for display.
+			.replace("\\_", "_");
```

Comment:
What is the primary function of this line? Is this supposed to match `\_` rather than `_`?

Reply:
Nevermind, I see you said this relates to #281.